libtransmission/torrent.cc

```diff
@@ -1000,9 +1000,9 @@ static time_t torrentGetIdleSecs(tr_torrent const* tor, tr_torrent_activity acti
         -1;
 }
 
-static inline bool tr_torrentIsStalled(tr_torrent const* tor, time_t idle_secs)
+static inline bool tr_torrentIsStalled(tr_torrent const* tor, size_t idle_secs)
 {
-    return tor->session->queueStalledEnabled() && idle_secs > tor->session->queueStalledMinutes() * 60;
+    return tor->session->queueStalledEnabled() && idle_secs > tor->session->queueStalledMinutes() * 60U;
```

Comment:
Kind of a wishy-washy answer, but it's because I'm a little amibivalent about adding a `time_t` VariantConverter. I'd kind of like to move away from `time_t` to `std::chrono`. It's not a high priority, but it's enough that I'm hesitant. :man_shrugging: